using_images/other_images/jenkins.adoc

```diff
@@ -514,6 +514,33 @@ an entrypoint. For more details about this, refer to the official
 https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly[Jenkins
 documentation].
 
+==== Permission considerations
+
+Consider for a moment in the above ConfigMap example the `<serviceAccount>` element of the 
```

Comment:
For the first paragraph:

In the previous ConfigMap example, the `<serviceAccount>` element of the 
Pod Template XML is the {product-title} Service Account used for the 
resulting Pod. The service account credentials mounted into the Pod, with permissions 
 associated with the service account, control which operations against the {product-title} master
are allowed from the Pod.